### DISCUSSION TITLE
Know Thy Self / Read Thy Self

### DISCUSSION THREAD

**Discussion by DanM3rcurius** [SecretDecks/Contentwriting#5] [Q&A] [answered]
## Feedback from reading of [first content piece](https://github.com/SecretDecks/Contentwriting/blob/main/Know-Thyself.md)

- [x] **change** title to: Read Thy Self

- [x]  **change:** Repetition of "know thyself" towards the end

- [x]  **reference** Ayhuasca/Psychedelics (in middle "origins" and "practical application")

- [x]  alphabetical order of people/religions if in comma separated list: change in first paragraph? or change wording

**Comment by stephen-rowan** [comment, marked as answer]
Suggestion - The "Know Thy Self / Read Thy Self" article series could also touch upon how SecretDecks intends to Know Itself. How the lessons of knowing and reading can be reflexive (apply to ourselves) as well as being exposition (applying in general). Although this shift to "getting to know ourselves" is more apparent in part two.

**Reply by evolayak** [reply]
Yes! @DanM3rcurius, could we use @stephen-rowan 's point to link the "Know thy self" 2 part article to the next article, introducing SecretDecks?

**Comment by evolayak** [comment]
I'm curious what everyone thinks about (very briefly, in a footnote) explaining why we use "thy self" instead of "thyself", since the latter is the common form...? Not only to avert any nit-picky comments from readers, but it could maybe make the brand stand out with this thoughtful and creative interpretation of a very ubiquitous phrase.

**Comment by DanM3rcurius** [comment]
Genius. Let’s do it.

**Comment by DanM3rcurius** [comment]
Same as why Jung talks of the Self.
Know Thy Self („know your own self“) refers to the holistic self knowledge (conscious and unconscious as well as divine parts), whereas know Thyself is the same as „know yourself“ and encompasses the conscious parts of the self (as I think to understand).

**Comment by DanM3rcurius** [comment]
Self in Jungian psychology
The Self in Jungian psychology is a dynamic concept which has undergone numerous modifications since it was first conceptualised as one of the Jungian archetypes. Historically, the Self, according to Carl Jung, signifies the unification of consciousness and unconsciousness in a person, and representing the psyche as a whole.It is realized as the product of individuation, which in his view is the process of integrating various aspects of one's personality. For Jung, the Self is an encompassing whole which acts as a container. It could be symbolized by a circle, a square, or a mandala.[Wikipedia](https://en.wikipedia.org/wiki/Self_in_Jungian_psychology)

https://en.wikipedia.org/wiki/Self_in_Jungian_psychology

**Reply by evolayak** [reply]
Brilliant. So now after reading your reply here I'd love to capitalise "self" in the text (aside from direct quotes), to adhere to Jung's concept of the Self. I know you already uploaded the articles to Medium, so I hope this is ok. What do you think? I'll go ahead and create a new branch and start a PR in the main repo so the changes are there if you want to accept them. (Still need to get a hang of editing in my repo 😅)

**Reply by DanM3rcurius** [reply]
Glad to hear it! The article on Medium is still in draft mode. I happily go in and change the "selfs" to "Selfs" 🧙🏿

**Reply by stephen-rowan** [reply]
Some materials on Jung and the Tarot you might find interesting ....

- [Tarot and the Archetypal Journey: The Jungian Path from Darkness to Light](https://www.amazon.co.uk/Tarot-Archetypal-Journey-Jungian-Darkness-ebook/dp/B07JD8DWN2/ref=tmm_kin_swatch_0?_encoding=UTF8&qid=&sr=)
- [Carl Jung and Tarot](https://marykgreer.com/2008/03/31/carl-jung-and-tarot/)

Jung was quite playful in his interpretations and experimented with personal mythology in his "Red Book". There are many Tarot parallels in Jung - like his notion of the shadow ("the dark side of your personality") - no doubt this also draws upon tropes in German Culture ("Du Doppelgänger! du bleicher Geselle!").

Also prominent is the notion of the journey. I think the Tarot is very Platonic, it reminds me a lot of Renaissance alchemical symbolism crossed with Dante. The alchemical part is often about reconciling opposites (finding the philosopher's stone) - whilst the Dante bits are about "Ascent as Descent" or losing ones Self to find One's Self (which is like the Shamanistic concept of self-disassembly - becoming a skeleton).

In this context capitalized "Self" becomes talismanic - it is a challenge to be present to your Self. Not simply resting (content) with yourself. Only after Dante has lost his mortal self and found his spiritual Self (like a place rather than an identity) - only then does his guide Virgil crown him "_Lord of yourself, I crown and mitre you_" and leaves him (Purgatorio, Canto 27. 139).

**Comment by DanM3rcurius** [comment]
What about naming the first content piece "Know Thy Self", the next one "Read Thy Self" and the one introducing the ACO "Read Our Self"? 
Or to keep the part i and part ii titles? Not too sure about this... @evolayak @stephen-rowan ?

**Reply by evolayak** [reply]
I like that! We close the first content piece with the suggestion to "read thy Self", so I think it's a logical flow to have that as the title for the second piece. I need to think a bit about the ACO intro - @DanM3rcurius, you mean reading ourselves as a collective and reading one another? Could be fun to play with it and have "read our Selves", but let em think a little bit.


We could then close content piece 2 with a mention of "reading our Selves" (I'm assuming that 
For the intro to the ACO,

**Reply by stephen-rowan** [reply]
Sorry just catching up with these comments now. I agree with @evolayak that there a logical flow from "Know Thy Self" to "Read Thy Self". 

"Know Thy Self" presents the big picture - my impression here is of a tension between egoistic self-knowledge (thinking that you do know yourself) and Socratic self-knowledge (knowing that you know nothing).  This distinction leads the "Know Thy Self" piece towards practice. And in Part II "Practical application" the focus of this practice is Shamanism.

So the second piece could focus on "Reading Your Self" or even "Reading Our Selves" (as a collective). It is interesting how we are using GitHub to read ourselves - the way we are able to comment, amend or even reverse each other's submissions. And we can time travel to different versions in the history. Through these iterations we should be able to incorporate some of SecretDeck's philosophy into our working practice.
